### ISSUE TITLE
interactive script doesnt really work on Arch

### ISSUE THREAD

**Issue by LiamDawe** [BenJetson/gnome-dash-fix#3]
So I tested the new interactive script and it doesn't really work.

Running it on Arch, it changes the cursor, but no prompts or anything come up. Click on anything and suddenly "os" and "sys" files appear in the directory.

Here's some output:
```
[liam@liam-antergos ~]$ '/home/liam/Downloads/gnome-dash-fix-0.2-beta/interactive.py' 
/home/liam/Downloads/gnome-dash-fix-0.2-beta/interactive.py: line 4: syntax error near unexpected token `('
/home/liam/Downloads/gnome-dash-fix-0.2-beta/interactive.py: line 4: `if sys.version_info < (3,0):'
```

**Comment by BenJetson** [comment]
Strange. Try these steps in order and see if it corrects the issue:
* Make sure package `python3` is installed.
* Flag the script as executable (shouldn't make a difference, but couldn't hurt)
* Does it work when you run the script as `python3 /home/liam/Downloads/gnome-dash-fix-0.2-beta/interactive.py`?

Let me know how it goes. :smiley:

**Comment by LiamDawe** [comment]
Yeah I have Python 3.6.1.
It is executable, would it even run if it wasn't?
Yes it works if I do: python3 /home/liam/Downloads/gnome-dash-fix-0.2-beta/interactive.py

**Comment by BenJetson** [comment]
Awesome! Since it's working, I'm going to close this issue. Feel free to open it back up if the issue reappears.

**Comment by BenJetson** [comment]
I think I found the source of this issue and have fixed it in the latest version (see commit a5bfdb9f033c620d7a677cb239afbf5047bcee67).
I modified the shebang to use `#!/usr/bin/env python3` instead.
